code/style.css

```diff
@@ -1,22 +1,28 @@
 body {
```

Comment:
Don't forget to add:
```
* {
  box-sizing: border-box;
}
```
This property can make building CSS layouts easier and a lot more intuitive. And the boxes will work like this by default:
width + padding + border = actual visible/rendered width of an element’s box
height + padding + border = actual visible/rendered height of an element’s box

😃